src/app/components/summary-actionbar/index.html

```diff
@@ -0,0 +1,237 @@
+<sky-demo-page
+  pageTitle="Summary action bar"
+  skyuxModule="SkySummaryActionBarModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The summary action bar component provides a docked container for actions and summary information related on a form or multiselect list. The <stache-code>sky-summary-action-bar</stache-code> element implements the summary action bar, and within this element, the <stache-code>sky-summary-action-bar-actions</stache-code> and <stache-code>sky-summary-action-bar-summary</stache-code> elements contain the actions and sumamry information respectively.
```

Comment:
Should we call out the individual action types here? Or is that overkill? We have them listed seperately below but don't mention them in this top block.

Reply:
We also don't mention the summary element and so the only place it is currently mentioned is the usage guidelines. We should probably mention that.